app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
This looks unused.

Reply:
It's used down in the propTypes variable to check the serializedData req. I could just use React.propTypes, but that's being depricated

Reply:
Is `SchoolAdministratorDashboards` still a React component? It doesn't inherit from React.Component or React.createClass anymore, so it looks like just a plain function to me. That makes me wonder if `serializedData` is still props, or just a plain argument to a function now?

Reply:
It is, actually! Anything that takes props as an argument and returns a React component is a valid component as far as React is concerned. https://reactjs.org/docs/components-and-props.html

Though you're right. It's also just a plain function with an argument. If we treat it like a react component we get type checking for the props and probably other stuff I'm just not thinking of atm.

Reply:
@edavidsonsawyer Ah I am learning so much here! This is cool!